lesson_02/job1/api.py

```diff
@@ -0,0 +1,28 @@
+API_URL = 'https://fake-api-vycpfa6oca-uc.a.run.app/'
```

Comment:
Imports should go first, 2 blank lines and constants, 2 blank lines and functions or classes. See PEP8 please